lib/model/query/forms.js

```diff
@@ -75,6 +75,9 @@ const createNew = (partial, project, publish = false) => async ({ run, Datasets,
     (partial.aux.key.isDefined() ? resolve(Option.none()) : getDataset(partial.xml)) // Don't parse dataset schema if Form has encryption key
   ]);
 
+  // Check that meta field (group containing instanceId and name) exists
+  await Forms.checkMeta(fields);
```

Comment:
do we need this in `createVersion`?

Reply:
I took it out of createVersion because of Matt's comment above: 

> What happens if you have a form that doesn't have a `meta` group, and you want to create a new form version that includes `meta`? Are you able to use Frontend to create a draft to do so, or will that request fail?

It would likely break the ability to create a new draft of any existing meta-less form. But at least catching the problem when the form is new is a good improvement.